types/manifest.d.ts

```diff
@@ -1,14 +1,14 @@
 declare module '*/raidboss_manifest.txt' {
   export interface RaidbossFileData {
-    [filename: string]: LooseTriggerSet;
+    [filename: string]: import('./trigger').LooseTriggerSet | string;
   }
   const raidbossFileData: RaidbossFileData;
   export default raidbossFileData;
 }
 
 declare module '*/oopsy_manifest.txt' {
   export interface OopsyFileData {
-    [filename: string]: LooseOopsyTriggerSet;
+    [filename: string]: import('./oopsy').LooseOopsyTriggerSet;
```

Comment:
Thank you for this syntax.  I wrestled with getting this import right in the past but could never figure it out.

Reply:
I was just trying, as I know `import()` can be used in JSDoc comment that typescript knows. Glad to see it works in code as well. 😁